devito/data/data.py

```diff
@@ -26,6 +27,9 @@ class Data(np.ndarray):
         If the i-th entry is True, then the i-th array dimension uses modulo indexing.
     allocator : MemoryAllocator, optional
         Used to allocate memory. Defaults to ``ALLOC_FLAT``.
+    distributor : Distributor, optional
+        The distributor attached to the original Function from which any subsequent
```

Comment:
in an ideal world, we avoid to talk about "Function" (and any other devito concept) in here